scripts/lib/spammer.ts

```diff
@@ -4,11 +4,11 @@ import { randomInt, sleep } from "./utils";
 import { Address, DataBusClient } from "../../client";
 
 const abi = [
-  "event MessageDepositV1(address indexed guardianAddress, (bytes32 depositRoot, uint256 nonce, uint256 blockNumber, bytes32 blockHash, bytes signature, uint256 stakingModuleId, (bytes32 version) app) data)",
-  "event MessagePauseV2(address indexed guardianAddress, (bytes32 depositRoot, uint256 nonce, uint256 blockNumber, bytes32 blockHash, bytes signature, uint256 stakingModuleId, (bytes32 version) app) data)",
-  "event MessagePauseV3(address indexed guardianAddress, (uint256 blockNumber, bytes signature, (bytes32 version) app) data)",
+  "event MessageDepositV1(address indexed guardianAddress, (bytes32 depositRoot, uint256 nonce, uint256 blockNumber, bytes32 blockHash, (bytes32 r, bytes32 vs) signature, uint256 stakingModuleId, (bytes32 version) app) data)",
```

Comment:
Optional:
Maybe use order as in deposit buffered ether method

```
        address guardianAddress,
        uint256 blockNumber,
        bytes32 blockHash,
        bytes32 depositRoot,
        uint256 stakingModuleId,
        uint256 nonce,
        Signature calldata signature,
        (bytes32 version) app,
```

Reply:
current order as in Council Daemon, do you think you should redo it?

Reply:
Done